### ISSUE TITLE
Top menu layout on Mobile

### ISSUE THREAD

**Issue by karlcow** [webcompat/webcompat.com#824]
![screenshot_2015-11-06-11-29-11](https://cloud.githubusercontent.com/assets/505230/10988132/0f136292-847d-11e5-8ac3-29dbd2525dce.jpg)
On Mobile the 3 icons on the top right are pushed I guess by the add-on message.
I would rather put the addon message on top and only once ;) or never :)

**Comment by magsout** [comment]
yes, We started discuss about it here https://github.com/webcompat/webcompat.com/issues/753

**Comment by karlcow** [comment]
Ah sorry I had forgotten.
duplicate of #753
